go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/hashicorp/raft-boltdb
 go 1.12
 
 require (
-	github.com/boltdb/bolt v1.3.1
+	// github.com/boltdb/bolt v1.3.1
```

Comment:
Did you intentionally leave this behind?

Reply:
whoops

Reply:
fixed!